test/TestExpectations.json

```diff
@@ -3258,6 +3258,13 @@
     "expectations": ["FAIL"],
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
+  {
+    "testIdPattern": "[launcher.spec] Launcher specs Puppeteer Puppeteer.launch should restore to original viewport size after taking fullPage screenshots when defaultViewport is null",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["cdp", "firefox"],
+    "expectations": ["FAIL"],
+    "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
```

Comment:
Could you please explain why it fails? I think it should not generally fail for Firefox here.

Reply:
I just check the failed test and see the suggestion that adding such code can avoid failure.
I will revert the code.

Reply:
Can you help to check why it is failed on firefox?

Reply:
Let me publish a change, I think it is okay to skip the test for Firefox CDP

Reply:
Thanks